paywall/src/paywall-script/index.ts

```diff
@@ -0,0 +1,12 @@
+import Postmate from 'postmate'
+
+const baseUrl = 'http://localhost:3000/checkout'
+const rawConfig = (window as any).unlockProtocolConfig
+const encodedConfig = encodeURI(JSON.stringify(rawConfig))
```

Comment:
These 3 values are the core functionality for setting up the connection to the child iframe. All we really need is a way to pick the base URL for a given environment and then it's all set.